cobalt/testing/BUILD.gn

```diff
@@ -0,0 +1,5 @@
+target(gtest_target_type, "cobalt_unittests") {
```

Comment:
This target doesn't really make sense as it is—it should either contain the `user_agent_platform_info_test.cc` source directly (if we want `cobalt_unittests` to be a runnable unit) or just be a group that depends on unit test targets like `user_agent_unittests` (if we just want it to be a helper target)